core/app/middleware_test.go

```diff
@@ -7,13 +7,72 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/flapflapio/simulator/internal/simtest"
 	"github.com/obonobo/mux"
 )
 
-func TestTrim(t *testing.T) {
-	type testcase struct{ name, input, output string }
```

Comment:
Why TestTrim was removed?

Reply:
It's still there, I just moved the test lower down in the same file - you can see it on line 72